aopy/preproc/wrappers.py

```diff
@@ -379,14 +470,28 @@ def proc_lfp(data_dir, files, result_dir, result_filename, overwrite=False, max_
 
         _, lfp_metadata = aodata.filter_lfp_from_broadband(broadband_filepath, result_filepath, dtype='int16', 
                                                     max_memory_gb=max_memory_gb, **filter_kwargs)
-    elif 'ecube' in files:
+        aodata.save_hdf(result_dir, result_filename, lfp_metadata, "/lfp_metadata", append=True)
+
+    elif 'ecube' in files and 'neuropixels' not in files:
```

Comment:
technically it is possible to have both a neuropixel probe and a ecube headstage recording at the same time. is there an easy way to change the logic to account for this?

Reply:
I'll add drive number for ecube so that we can deal with simultaneous recording